<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1179</width>
    <height>763</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MyInfo</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
    background-color: cadetblue;
    border: 2px solid black;
    
   
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QWidget{
    background-color: cadetblue;
    
}</string>
   </property>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1231</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>36</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: rgb(255, 240, 231);
border: 2px solid black;
}</string>
    </property>
    <property name="text">
     <string>MySQL Connections</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>80</y>
      <width>1081</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame{
border: 2px solid rgb(255, 240, 231);
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="barcode_button">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>80</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
border: 0px;
color: rgb(255, 240, 231)
}</string>
     </property>
     <property name="text">
      <string>Bar Code</string>
     </property>
    </widget>
    <widget class="QPushButton" name="search_button">
     <property name="geometry">
      <rect>
       <x>980</x>
       <y>10</y>
       <width>80</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
color: rgb(255, 240, 231);
border: 2px solid black
}

QPushButton:hover{
border: 2px solid rgb(0, 85, 255);
}</string>
     </property>
     <property name="text">
      <string>Search</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="barcode_line">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>9</y>
       <width>441</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
background-color: rgb(255, 240, 231);
border : 1px solid black;
}

QLineEdit:focus{
background-color: white;
}</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="search_line">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>9</y>
       <width>371</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{
background-color: rgb(255, 240, 231);
border : 1px solid black;
}

QLineEdit:focus{
background-color: white;
}</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="membid_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>99</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border: 0px;
color: rgb(255, 240, 231)
}</string>
    </property>
    <property name="text">
     <string>Member ID</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="id_line">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>170</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
background-color: rgb(255, 240, 231);
border : 1px solid black;
}

QLineEdit:focus{
background-color: white;
}</string>
    </property>
   </widget>
   <widget class="QPushButton" name="firstname_button">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>170</y>
      <width>94</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border: 0px;
color: rgb(255, 240, 231)
}</string>
    </property>
    <property name="text">
     <string>First name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="firstname_line">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>170</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
background-color: rgb(255, 240, 231);
border : 1px solid black;
}

QLineEdit:focus{
background-color: white;
}</string>
    </property>
   </widget>
   <widget class="QPushButton" name="surname_button">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>170</y>
      <width>79</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border: 0px;
color: rgb(255, 240, 231)
}</string>
    </property>
    <property name="text">
     <string>Surname</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="surname_line">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>170</y>
      <width>70</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
background-color: rgb(255, 240, 231);
border : 1px solid black;
}

QLineEdit:focus{
background-color: white;
}</string>
    </property>
   </widget>
   <widget class="QPushButton" name="address_button">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>170</y>
      <width>82</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border: 0px;
color: rgb(255, 240, 231)
}</string>
    </property>
    <property name="text">
     <string> Address </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="address_line">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>170</y>
      <width>95</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
background-color: rgb(255, 240, 231);
border : 1px solid black;
}

QLineEdit:focus{
background-color: white;
}</string>
    </property>
   </widget>
   <widget class="QPushButton" name="email_button">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>170</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border: 0px;
color: rgb(255, 240, 231)
}</string>
    </property>
    <property name="text">
     <string>   Email    </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="email_line">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>170</y>
      <width>95</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
background-color: rgb(255, 240, 231);
border : 1px solid black;
}

QLineEdit:focus{
background-color: white;
}</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pobox_button">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>170</y>
      <width>70</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border: 0px;
color: rgb(255, 240, 231)
}</string>
    </property>
    <property name="text">
     <string>PO Box </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="pobox_line">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>170</y>
      <width>95</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777201</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
background-color: rgb(255, 240, 231);
border : 1px solid black;
}

QLineEdit:focus{
background-color: white;
}</string>
    </property>
   </widget>
   <widget class="QPushButton" name="gender_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>66</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border: 0px;
color: rgb(255, 240, 231)
}</string>
    </property>
    <property name="text">
     <string>Gender</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="mobile_line">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>230</y>
      <width>281</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
background-color: rgb(255, 240, 231);
border : 1px solid black;
}

QLineEdit:focus{
background-color: white;
}</string>
    </property>
   </widget>
   <widget class="QPushButton" name="type_button">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>230</y>
      <width>52</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border: 0px;
color: rgb(255, 240, 231)
}</string>
    </property>
    <property name="text">
     <string>Type</string>
    </property>
   </widget>
   <widget class="QComboBox" name="type_combobox">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>230</y>
      <width>281</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox{
background-color: rgb(255, 240, 231);
border : 1px solid black;
}

QComboBox:focus{
background-color: white;
}</string>
    </property>
    <item>
     <property name="text">
      <string>Annual Member</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Quarterly</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Montly</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="gender_combobox">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>230</y>
      <width>291</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox{
background-color: rgb(255, 240, 231);
border : 1px solid black;
}

QComboBox:focus{
background-color: white;
}</string>
    </property>
    <item>
     <property name="text">
      <string>Male</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Female</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="addnew_button">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>690</y>
      <width>169</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
color: rgb(255, 240, 231);
border: 2px solid black;
background-color: rgb(255, 240, 231);
color: cadetblue;
}

QPushButton:hover{
border: 2px solid rgb(0, 85, 255);
background-color: white;
color: black;
}</string>
    </property>
    <property name="text">
     <string>Add New</string>
    </property>
   </widget>
   <widget class="QPushButton" name="showrecord_button">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>690</y>
      <width>169</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
color: rgb(255, 240, 231);
border: 2px solid black;
background-color: rgb(255, 240, 231);
color: cadetblue;
}

QPushButton:hover{
border: 2px solid rgb(0, 85, 255);
background-color: white;
color: black;
}</string>
    </property>
    <property name="text">
     <string>Show Record</string>
    </property>
   </widget>
   <widget class="QPushButton" name="update_button">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>690</y>
      <width>169</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
color: rgb(255, 240, 231);
border: 2px solid black;
background-color: rgb(255, 240, 231);
color: cadetblue;
}

QPushButton:hover{
border: 2px solid rgb(0, 85, 255);
background-color: white;
color: black;
}</string>
    </property>
    <property name="text">
     <string>Update</string>
    </property>
   </widget>
   <widget class="QPushButton" name="delete_button">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>690</y>
      <width>169</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
color: rgb(255, 240, 231);
border: 2px solid black;
background-color: rgb(255, 240, 231);
color: cadetblue;
}

QPushButton:hover{
border: 2px solid rgb(0, 85, 255);
background-color: white;
color: black;
}</string>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QPushButton" name="reset_button">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>690</y>
      <width>169</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
color: rgb(255, 240, 231);
border: 2px solid black;
background-color: rgb(255, 240, 231);
color: cadetblue;
}

QPushButton:hover{
border: 2px solid rgb(0, 85, 255);
background-color: white;
color: black;
}</string>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exit_button">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>690</y>
      <width>169</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
color: rgb(255, 240, 231);
border: 2px solid black;
background-color: rgb(255, 240, 231);
color: cadetblue;
}

QPushButton:hover{
border: 2px solid rgb(0, 85, 255);
background-color: white;
color: black;
}</string>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="mobile_button">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>230</y>
      <width>66</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border: 0px;
color: rgb(255, 240, 231)
}</string>
    </property>
    <property name="text">
     <string>Mobile</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>281</y>
      <width>1151</width>
      <height>391</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QListWidget{
background-color: rgb(255, 240, 231);
border : 1px solid black;
}

QLineEdit:focus{
background-color: white;
}</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1179</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
